packages/cloudflare/src/api/memory-queue.spec.ts

```diff
@@ -3,8 +3,8 @@ import { beforeAll, describe, expect, it, vi } from "vitest";
 
 import cache from "./memory-queue";
 
-vi.mock("./.next/prerender-manifest.json", () => ({
```

Comment:
This mock would fail to resolve a value for the dynamic import intermittently in the tests, so I moved the function to its own file so that the tests can be stable.

<img width="1102" alt="image" src="https://github.com/user-attachments/assets/2e247712-4e18-4b16-8ffa-1c3b699443ee" />

Reply:
it appears to be when two calls to the dynamic import happen at the exact same time that vitest fails to resolve them, so managed to fix this by changing the test so we're not calling it twice at the same time...